packages/core/core/src/Parcel.js

```diff
@@ -275,10 +275,13 @@ export default class Parcel {
         shouldBuildLazily: options.shouldBuildLazily,
         requestedAssetIds: this.#requestedAssetIds,
       }); // ? should we create this on every build?
+
       let {
         assetGraph,
         changedAssets,
         assetRequests,
+        previousAssetGraphHash,
+        shouldBundle,
```

Comment:
Something we've briefly talked about, but double checking that are we okay with making this an optimization without the "--incremental" flag ?